web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/BasicInformation/index.tsx

```diff
@@ -101,13 +101,17 @@ export const BasicInformation = (props: {
     [appMetaData?.id, refetchAppMetadata],
   );
 
-  const { url, showDeveloperFlag } = useMemo(() => {
+  // Show QR quick action if the app has an integration URL
+  // New mini apps may have an empty integration URL
+  const showQrQuickAction = !!appMetaData?.integration_url;
```

Comment:
We would want to count empty string as false here. I believe !! accepts ""
```suggestion
  const showQrQuickAction = Boolean(appMetaData?.integration_url);
```

Reply:
It works with `!!` as well! Changing to Boolean for more clarity.